src/rpc.rs

```diff
@@ -87,6 +95,20 @@ impl Mission for RPC {
     }
 }
 
+fn to_status(err: dcs_module_ipc::Error) -> Status {
+    use dcs_module_ipc::Error;
+    match err {
+        Error::Script { kind, message } => match kind.as_deref() {
+            Some("INVALID_ARGUMENT") => Status::invalid_argument(message),
+            Some("NOT_FOUND") => Status::not_found(message),
+            Some("ALREADY_EXISTS") => Status::already_exists(message),
+            Some("UNIMPLEMENTED") => Status::unimplemented(message),
```

Comment:
I've only mapped some of the GRPC types. Basically only the ones that I thought could make sense in the future.